.github/workflows/build_and_test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.12", "3.13"]
```

Comment:
I recommend that we do not update to supporting Python 3.13 and 3.12 until Mar.-Apr. 2026 when Python 3.13 is scheduled to come out of "bugfix" mode (see "[Status of Python Versions](https://devguide.python.org/versions/)" page). I made a proposal today of a "Python version policy" for OG-Core and its country calibration repositories in [this OG-Core discussion](https://github.com/PSLmodels/OG-Core/discussions/1046). I would love to hear your thoughts on this.

Reply:
OK. Let's move to Python 3.12 and 3.13. @jdebacker